src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ErrorTests.kt

```diff
@@ -0,0 +1,15 @@
+package org.imperial.mrc.mint.integration.endpoints
+
+import org.assertj.core.api.AssertionsForClassTypes.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.web.client.getForEntity
+
+class ErrorTests: EndpointTests() {
```

Comment:
I was going to say should probably  test the 500 case too, but then couldn't think of a way to force a 500: I modified the code to self-sabotage to test it manually. Could maybe write a test with a self-defeating config, e.g. wrong mintr endpoint, I guess that would probably throw an exception?

Reply:
I'm not too worried given the unit tests cover the 500 case?

Reply:
Although I suppose the "500.ftl" template could be missing in theory, ok I'll figure something out.